controllers/database_reconciler_helpers.go

```diff
@@ -70,15 +70,19 @@ func (r *DatabaseReconciler) addFinalizer(ctx context.Context, req ctrl.Request,
 func (r *DatabaseReconciler) handleDelete(ctx context.Context, database *ndbv1alpha1.Database, ndbClient *ndb_client.NDBClient) (ctrl.Result, error) {
 	log := ctrllog.FromContext(ctx)
 	log.Info("Database CR is being deleted.")
+	r.recorder.Event(database, "Normal", "DatabaseCRDeletionStarted", "Database CR is being deleted.")
```

Comment:
we should change it to "Database is being deleted.", no? "CR" is an internal terminology. I don't think k8s implementation reveals "CR" terminology to the outside world.

Reply:
I discussed this in the standup today and we agreed that we will have to use some Kubernetes related terms to make events really useful for troubleshooting. If we abstract it out too much it would become confusing. Event in-built k8s events have some specific errors.